### ISSUE TITLE
Support of parameterized test

### ISSUE THREAD

**Issue by NitroTamagotschi** [fundakol/pytest-jira-xray#96]
It would be nice if parameterized tests would be supported as well.
So that tests which got executed in multiple iterations with different parameters as [documented here](https://docs.getxray.app/display/XRAY/Parameterized+Tests).
An example you'll find below under key **iterations**
```
	"id": 10603,
	"status": "EXECUTING",
	"color": "#F1E069",
	"testKey": "ID-59",
	"testExecKey": "ID-60",
	"assignee": "User1",
	"executedBy": "User1",
	"startedOn": "2023-11-16T08:00:21+01:00",
	"startedOnIso": "2023-11-16T08:00:21+01:00",
	"defects": [],
	"evidences": [],
	"testEnvironments": [],
	"fixVersions": [],
	"customFields": [],
	"iterations": [
		{
			"id": 3,
			"status": "PASS",
			"color": "#95C160",
			"parameters": [
				{
					"name": "Param1",
					"value": "1"
				},
				{
					"name": "Param2",
					"value": "A"
				}
			]
		},
		{
			"id": 4,
			"status": "TODO",
			"color": "#A2A6AE",
			"parameters": [
				{
					"name": "Param1",
					"value": "2"
				},
				{
					"name": "Param2",
					"value": "B"
				}
			]
		}
	]
}
```

**Comment by fundakol** [comment]
JSON format used to import Test results does not have `iterations` field. 
https://docs.getxray.app/display/XRAY/v2.0#/Import/post-import-execution

**Comment by NitroTamagotschi** [comment]
Yes you're right. Its only supported for manual tests until today
![image](https://github.com/fundakol/pytest-jira-xray/assets/48655006/ca301d79-b00a-4a26-92e0-d2d39abbf6b9)

**Comment by akyekth** [comment]
Hi i can able to import results to jira by using below pytest hook.
test_input.json has test_key: "JIRA-1"
my tests created through parameterized /pytest_generate_tests hook

conftest.py:
------------------------------
def pytest_runtest_call(self, item):
        inputdict= item.callspec.params["test_case"]
        test_key=inputdict.test_key
        if not test_key == "":
            item.add_marker(pytest.mark.xray(f'{test_key}'))
